Pipeliner/Common/Result.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Result.swift
+//  Pipeliner
+//
+//  Created by dx hero on 11/1/20.
+//
+
+import Foundation
+
+enum Result<T> {
```

Comment:
You might just use [Swift.Result](https://developer.apple.com/documentation/swift/result) instead of introducing a new enum